### DISCUSSION TITLE
basecoding

### DISCUSSION THREAD

**Discussion by Mateo-Johnson** [BroncBotz3481/YAGSL-Example#91] [General]
Hey, does anybody know what is needed to get this to be a basic working robot, using sparkmaxes and a navx? All the best

**Comment by thenetworkgrinch** [comment]
You should checkout the based example here https://github.com/BroncBotz3481/YAGSL-Example/tree/main/src/main/deploy/swerve/neo

**Reply by Mateo-Johnson** [reply]
yeah, sorry to not have clarified, thats what I was discussing, asking if there was anything that was missing, or anything that needed to be configured in that example
